[2025-05-07T02:25:20.993+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_load.wait_clients_csv manual__2025-05-07T02:25:14.827420+00:00 [queued]>
[2025-05-07T02:25:20.998+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_load.wait_clients_csv manual__2025-05-07T02:25:14.827420+00:00 [queued]>
[2025-05-07T02:25:20.998+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2025-05-07T02:25:21.007+0000] {taskinstance.py:1327} INFO - Executing <Task(FileSensor): wait_clients_csv> on 2025-05-07 02:25:14.827420+00:00
[2025-05-07T02:25:21.011+0000] {standard_task_runner.py:57} INFO - Started process 4421 to run task
[2025-05-07T02:25:21.013+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'extract_load', 'wait_clients_csv', 'manual__2025-05-07T02:25:14.827420+00:00', '--job-id', '231', '--raw', '--subdir', 'DAGS_FOLDER/extract_load.py', '--cfg-path', '/tmp/tmpmpj29twa']
[2025-05-07T02:25:21.014+0000] {standard_task_runner.py:85} INFO - Job 231: Subtask wait_clients_csv
[2025-05-07T02:25:21.025+0000] {logging_mixin.py:150} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-07T02:25:21.049+0000] {task_command.py:410} INFO - Running <TaskInstance: extract_load.wait_clients_csv manual__2025-05-07T02:25:14.827420+00:00 [running]> on host 5c5d1775d94e
[2025-05-07T02:25:21.106+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='finbest' AIRFLOW_CTX_DAG_ID='extract_load' AIRFLOW_CTX_TASK_ID='wait_clients_csv' AIRFLOW_CTX_EXECUTION_DATE='2025-05-07T02:25:14.827420+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-07T02:25:14.827420+00:00'
[2025-05-07T02:25:21.121+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2025-05-07T02:25:21.122+0000] {filesystem.py:64} INFO - Poking for file /data/clients.csv
[2025-05-07T02:25:21.122+0000] {filesystem.py:69} INFO - Found File /data/clients.csv last modified: 20250501214538
[2025-05-07T02:25:21.122+0000] {base.py:255} INFO - Success criteria met. Exiting.
[2025-05-07T02:25:21.130+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=extract_load, task_id=wait_clients_csv, execution_date=20250507T022514, start_date=20250507T022520, end_date=20250507T022521
[2025-05-07T02:25:21.145+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-05-07T02:25:21.163+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
